web/src/global.d.ts

```diff
@@ -0,0 +1,10 @@
+import "./config";
+
+declare global {
+  interface Window {
+    REEARTH_E2E_ACCESS_TOKEN?: string;
+    REEARTH_E2E_CESIUM_VIEWER?: Viewer;
+    React?: any;
+    ReactDOM?: any;
```

Comment:
Do we need to expose these for CMS? They are exposed in reearth-web now for the extension system, but unless there is a reason for them in CMS, they don't need to be exposed here.